src/client/components/UserPage/Drawer/ControlPanel/DrawerHeader.tsx

```diff
@@ -13,21 +13,15 @@ import CopyButton from '../../Widgets/CopyButton'
 const useStyles = makeStyles((theme) =>
   createStyles({
     drawerTitleDiv: {
-      display: 'flex',
-      alignItems: 'flex-end',
-      justifyContent: 'space-between',
       marginBottom: theme.spacing(6.5),
       marginTop: theme.spacing(1.5),
       [theme.breakpoints.up('md')]: {
         alignItems: 'center',
         marginTop: 0,
       },
     },
-    copyLinkDiv: {
-      display: 'flex',
-    },
-    copyIcon: {
-      marginRight: 5,
+    copyButtonWrapper: {
+      marginLeft: -8,
```

Comment:
Could we avoid the use of negative margins?

Reply:
The `CopyButton` component (http://localhost:3000/test-google) has a 8px padding. The use of negative margin is to align the `CopyButton` with other elements as per design. 

If the elements do not need to be visually aligned, we can remove the negative margin. Otherwise, I think it is preferred to have negative margin rather than to restructure the layout to achieve the design.

Without negative margin
![image](https://user-images.githubusercontent.com/6182649/87306554-c2857780-c54a-11ea-96e4-418fd5bd539d.png)

With negative margin,
![image](https://user-images.githubusercontent.com/6182649/87306377-818d6300-c54a-11ea-834f-5bf2acabc2a1.png)

With negative margin, hovered state
![image](https://user-images.githubusercontent.com/6182649/87306406-8d792500-c54a-11ea-93fd-2147a6b371e0.png)